From fb8f71e4aaafe559f6e9a4370b3858dd05786e53 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 25 May 2016 21:42:57 -0400
Subject: [PATCH 30/62] enable gui pages

- support smart repository setting
- support software selection setting
- support timezone setting
- support user creation setting

Upstream-Status: Inappropriate [wrlinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/ui/gui/hubs/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/ui/gui/hubs/__init__.py b/pyanaconda/ui/gui/hubs/__init__.py
index 2bc85ac..0e1bb4c 100644
--- a/pyanaconda/ui/gui/hubs/__init__.py
+++ b/pyanaconda/ui/gui/hubs/__init__.py
@@ -217,7 +217,7 @@ class Hub(GUIObject, common.Hub):
         log.info("_updateCompleteness %s" % (spoke.mainWidgetName))
         # It is minimal installation for now, the non-implemented
         # functions will be disabled.
-        if spoke.mainWidgetName not in ["storageWindow", "passwordWindow"]:
+        if spoke.mainWidgetName not in ["storageWindow", "passwordWindow", "sourceWindow", "softwareWindow", "datetimeWindow", "userCreationWindow"]:
             spoke.selector.set_sensitive(False)
         else:
             spoke.selector.set_sensitive(spoke.sensitive and spoke.ready)
-- 
2.7.4

